### PR TITLE
Nrirdnbtrsy patch 1

### PR DESCRIPTION
**Pull request by nrirdnbtrsy**

### CODE REVIEW

**Comment by nrirdnbtrsy** [comment]
ada?

File: irdnatest.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# sine (rs)
```
Comment (SyafYus) [inline irdnatest.md]: no need to put the #sine(rs) because we want all our profile to look similar

File: irdnatest.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+name | registration number | hobby | fav math concept | other info | photo | link
```
Comment (SyafYus) [inline irdnatest.md:1]: make this as the first line

**Review by SyafYus — CHANGES_REQUESTED** [review]
just a little change should be fine

File: irdnatest.md (hunk 1 above)
Comment (nrirdnbtrsy) [reply irdnatest.md]: Ok

**Review by SyafYus — COMMENTED** [review]
ok noice

**Pull request merged**